internal/api/handlers/requestadapters/schedulers_test.go

```diff
@@ -549,6 +549,7 @@ func TestFromApiCreateSchedulerRequestToEntity(t *testing.T) {
 							},
 						},
 					},
+					Annotations: []*api.Annotation{},
```

Comment:
Yes. Most of the schedulers won't have annotations. But this will allow for those who want to add annotations. I've added the annotations in some tests, but to be honest I don't know how to test it since the main change happens in the runtime.  When we apply the annotation to the game rooms I think will be possible to add tests. WDYT?